packages/mini-browser/src/common/mini-browser-endpoint.ts

```diff
@@ -0,0 +1,26 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/**
+ * The mini-browser can now serve content on its own host/origin.
+ *
+ * The virtual host can be configured with this `THEIA_MINI_BROWSER_HOST_PATTERN`
+ * environment variable. Only a `{{hostname}}` variable is allowed and will be replaced.
+ */
+export namespace MiniBrowserEndpoint {
+    export const HOST_PATTERN_ENV = 'THEIA_MINI_BROWSER_HOST_PATTERN';
+    export const HOST_PATTERN_DEFAULT = 'mini-browser.{{hostname}}';
```

Comment:
We should include uuid here, i.e. you have one preview which stores some credentials in local storage when another preview can steal it. Previews should be isolated from each other as well.

Reply:
Ok